release-notes/5.0/api-diff/netstandard2.1/5.0_System.Resources.md

```diff
@@ -0,0 +1,11 @@
+# System.Resources
+
+``` diff
+ namespace System.Resources {
```

Comment:
@ericstj is it expected to see this diff here?

Reply:
Yes